tfsdk/serve_test.go

```diff
@@ -1738,10 +1739,47 @@ func TestServerPlanResourceChange(t *testing.T) {
 				"favorite_colors": tftypes.NewValue(tftypes.List{ElementType: tftypes.String}, []tftypes.Value{
 					tftypes.NewValue(tftypes.String, "red"),
 					tftypes.NewValue(tftypes.String, "orange"),
+					tftypes.NewValue(tftypes.String, "yellow"),
```

Comment:
This is a test fix since the proposed new state contained it, so the planned state would also contain it.